src/main/resources/events-mapping.json

```diff
@@ -1,114 +1,49 @@
 {
   "properties": {
-    "action_name": {
-      "type": "keyword"
-    },
-    "user_id": {
-      "type": "keyword"
-    },
-    "query_id": {
-      "type": "keyword"
-    },
-    "page_id": {
-      "type": "keyword"
-    },
-    "message": {
-      "type": "keyword"
-    },
-    "message_type": {
-      "type": "keyword"
-    },
+    "application": { "type": "keyword", "ignore_above": 256 },
+    "action_name": { "type": "keyword", "ignore_above": 100 },
```

Comment:
should we roll these into the spec then as well?

Reply:
I thought they were???